src/app/content/components/constants.ts

```diff
@@ -5,6 +5,8 @@ export const bookBannerDesktopMiniHeight = 7;
 export const bookBannerMobileBigHeight = 10.4;
 export const bookBannerMobileMiniHeight = 6;
 
+export const scrollOffset = 4;
```

Comment:
Might be better to set it to `3`. It will prevent it from showing a bit of the text above the figure

`.os-figure {
      margin: 3rem 0;
    }`